doc/source/reading_and_writing.rst

```diff
@@ -0,0 +1,193 @@
+.. -*- coding: utf-8 -*-
+.. _reading_and_writing:
+
+=========================
+Reading and writing files
+=========================
+
+Input
+================
+
+Read information from topology and coordinate files through the :ref:`Universe <universe-loading-label>`. MDAnalysis supports a number of :ref:`formats <formats-label>`. These formats can come with format-specific keyword arguments, such as the :ref:`LAMMPS DATA <DATA-label>` ``atom_style`` specification.
+
```

Comment:
I would put a basic example here before diving into ChainReader etc, something like
```
u = mda.Universe(TOPOLOGY, "trajectory.xtc")
```
to load a Gromacs XTC or
```
u = mda.Universe(TOPOLOGY, "trajectory.dcd")
```
for a CHARMM/NAMD DCD file. Note that the 'file format' (link to Trajectory formats) is automatically detected (based on the file extension).

Reply:
Assume that someone skipped the Quick Start and went right for what they wanted to do "Reading Trajectories".

Reply:
That does sound like a lot of people, myself included.